src/common/procedure/src/store.rs

```diff
@@ -269,15 +274,21 @@ mod tests {
             step: 2,
             key_type: KeyType::Step,
         };
-        assert_eq!(format!("{procedure_id}/0000000002.step"), key.to_string());
+        assert_eq!(
+            format!("{KEY_PREFIX}{procedure_id}/0000000002.step"),
```

Comment:
Can we add a macro such as `proc_path!("{procedure_id}/0000000002.step")` to wrap these `format!` invocations? It's more straightforward and maintainable.